pkg/gce-pd-csi-driver/node.go

```diff
@@ -81,6 +85,8 @@ const (
 	fsTypeExt3                 = "ext3"
 
 	readAheadKBMountFlagRegexPattern = "^read_ahead_kb=(.+)$"
+
+	deviceInUseTimeout = 30
```

Comment:
Can you make this configurable with a feature flag? We may need to reduce this if 30 seconds is too high.

Reply:
Also, consider making this a duration, so we have better type safety.